hotelReservation/helm-chart/hotelreservation/charts/mongodb-rate/values.yaml

```diff
@@ -12,3 +12,7 @@ container:
   name: hotel-reserv-rate-mongo
   ports:
   - containerPort: 27017
+
+useAccessControl: true
+useInitScript: true
```

Comment:
These should default false.  Like this, all legacy configurations will suddenly start deploying with the new feature enabled.